frontend/components/site/siteBuildLogs.js

```diff
@@ -38,7 +34,7 @@ class SiteBuildLogs extends React.Component {
       return <LoadingIndicator />;
     }
 
-    if (!buildLogs || !buildLogs.data || !buildLogs.data.length) {
+    if (!buildLogs || !buildLogs.data || !buildLogs.data.length || buildLogs.data.length === 0) {
```

Comment:
This part 
`|| buildLogs.data.length === 0` shouldn't be necessary as `!buildLogs.data.length` covers the `0` case.

Reply:
is it possible this build could  have 0 logs and there are no logs to display?